submission/example_method.py

```diff
@@ -1,10 +1,15 @@
 """ Demo: Creates a simple new method and applies it to a single CL setting.
+
```

Comment:
IDK how I feel about the renaming. This is clearly meant for the SL track, right? So this method can't (out of the box) solve the RL task or both, right? So it should be named accordingly, right?

Reply:
True that!